pkg/importer/dataStream.go

```diff
@@ -428,6 +446,14 @@ func (d *DataStream) tarReader() (io.Reader, int64, error) {
 	return tr, hdr.Size, nil
 }
 
+func (d *DataStream) mulFileTarReader() (io.Reader, int64, error) {
```

Comment:
Since we don't have an impl here yet, if you introduce the wrapper as I mentioned above it could just return nil,0,nil for this type without adding/calling a function with no implementation.